docs/query_proxy.md

```diff
@@ -0,0 +1,205 @@
+# Operating the Wormhole Queries Proxy Server
+
+The Wormhole queries proxy server (sometimes referred to as the CCQ proxy) is a server that listens on a
+REST endpoint for Wormhole query requests. It validates those requests and forwards them to the guardian
+CCQ P2P network for processing by the guardians. It then accumulates the responses from the guardians,
+verifies quorum and forwards the response to the client.
+
+## Building the Proxy Server
+
+The proxy server runs as another instance of the `guardiand` process, similar to the spy. It is built exactly
+the same as the spy, and requires the same dependencies. Please see the [Operations Guide](operations.md) for
+details on how to build `guardiand`.
+
+## Deploying the Proxy Server
+
+The proxy server can be deployed just like the spy, including potentially running in a container. Note that it
+requires a public IP address to listen for REST requests, and it needs to be able to reach the guardian P2P network.
+
+The proxy is not particularly resource intensive, so should run successfully on a reasonable size VM.
+
+## Configuring the Proxy Server
+
+There are two main parts to configuring the proxy server. The first is setting up the command line arguments,
+which generally will not change after initial setup. The second part of the configuration is the permissions file,
+which will change as the requirements of integrators change.
+
+### Proxy Server Command Line Arguments
+
+The following is a sample command line for running the proxy server in mainnet.
+
+```code
+
+build/bin/guardiand query-server \
+      --env "mainnet" \
+      --nodeKey /home/ccq/data/ccq_server.nodeKey \
+      --permFile "/home/ccq/data/ccq_server.perms.json" \
+      --signerKey "/home/ccq/data/ccq_server.signerKey" \
+      --listenAddr "[::]:80" \
```

Comment:
Listening on port 80 requires this to run as root. Do we want to advertise that as the default?